CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 - use `AbstractRule` instead of `Rule` in `_rules_from_strategy`
+- fix `CombinatorialSpecification.share()` which was missing
```

Comment:
We should probably only have something in the `### Added` section about the new share functionality

Reply:
It is under 2.1.0. I will add a note here that we added it for forests too.